pbs/README.md

```diff
@@ -0,0 +1,57 @@
+# PBS
+
+Proposer builder seperation for Op Stack.
+
+## How to run the devnet
+
+You first need to build the op-geth docker image with builder API support.
+
+```shell
+$ git clone git@github.com:flashbots/op-geth.git
+$ cd op-geth
+$ docker build . -t <YOUR_OP_GETH_BUILDER_IMAGE>
+```
+
+Then you can run the devnet with the following command:
+
+```shell
+$ BUILDER_IMAGE=<YOUR_OP_GETH_BUILDER_IMAGE> DEVNET_BUILDER=true make devnet-up
+```
+
+To additionally enable load testing through [tx-fuzz](https://github.com/MariusVanDerWijden/tx-fuzz), you can run the following command:
+
+```shell
+$ BUILDER_IMAGE=<YOUR_OP_GETH_BUILDER_IMAGE> DEVNET_LOAD_TEST=true DEVNET_BUILDER=true make devnet-up
+```
+
+If the `BUILDER_IMAGE` is not set, the devnet will use the image from `flashbots/op-geth:latest`.
+
+## Configuration
+
+These are the configuration options to enable PBS for the devnet.
+
+### Sequencer
+
+There are three flags that congifure the sequencer to request payloads from builder API endpoint:
+
+- `l2.builder.enabled`: Enable the builder API request to get payloads built from the builder.
+- `l2.builder.endpoint`: The URL of the builder API endpoint.
```

Comment:
This endpoint should follow https://github.com/ethereum-optimism/specs/pull/116/files#diff-3424f00d095fe96d07f55b760552c1555a6550c7417a375a7027cae4e51083a6R88 where the sequencer has a predefined set of builder pub keys and endpoints. 
The pubkey can be embedded in the url like the relay endpoint e.g. `https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net`